test/unit/cinstance/trial_test.rb

```diff
@@ -38,17 +38,18 @@ class TrialPeriodPlanTest < ActiveSupport::TestCase
     end
 
     test 'should compute correct expiration date' do
-      Timecop.freeze(2009,11,4) do
+      travel_to(Date.new(2009,11,4)) do
         cinstance = FactoryBot.create(:cinstance, plan: @plan)
         expected = (cinstance.created_at + 30.days).to_date
         assert_equal expected, cinstance.trial_period_expires_at.to_date
       end
     end
 
     test 'should expire the trial after 31 days' do
-      Timecop.freeze
+      freeze_time
```

Comment:
Do you think we can drop `freeze_time` and `travel_back`?
I don't see what value they add, I think just `travel_to` after the creation should work fine.

Reply:
or use a block if needed

Reply:
Yeah, it's not needed